README.md

```diff
@@ -92,18 +92,48 @@ Here are our six core certifications:
 - [Advanced Node and Express](https://learn.freecodecamp.org/information-security-and-quality-assurance/advanced-node-and-express)
   <br />
   <br />
-  **Projects**: Metric-Imperial Converter, Issue Tracker, Personal Library, Stock Price Checker, Anonymous Message Board
+  **Projects**: Metric-Imperial Converter, Issue Tracker, Personal Library, Sudoku Solver, American British Translator
+   
+#### 7. Scientific Computing with Python Certification
 
-#### Full Stack Development Certification
+- [Introduction to Python for Everybody](https://learn.freecodecamp.org/scientific-computing-with-python/python-for-everybody)
+  <br />
+  <br />
+  **Projects**: Arithmetic Formatter, Time Calculator, Budget App, Polygon Area Calculator, Probability Calculator
+
+#### 8. Data Analysis with Python Certification
+
+- [Data Analysis with Python Course](https://learn.freecodecamp.org/data-analysis-with-python/data-analysis-with-python-course)
+- [NumPy](https://learn.freecodecamp.org/data-analysis-with-python/numpy)
+  <br />
+  <br />
+  **Projects**: Mean-Variance-Standard Deviation Calculator, Demographic Data Analyzer, Medical Data Visualizer, Page View Time Series Visualizer, Sea Level Predictor
+  
+#### 9. Information Security Certification
+
+- [Information Security with HelmetJS](https://learn.freecodecamp.org/information-security/information-security-with-helmetjs)
+- [Python for Penetration Testing](https://learn.freecodecamp.org/information-security/python-for-penetration-testing)
+  <br />
+  <br />
+  **Projects**: Stock Price Checker, Anonymous Message Board, Port Scanner, SHA-1 Password Cracker, Secure Real Time Multiplayer Game 
+  
+#### 10. Machine Learning with Python Certification
+
+- [TensorFlow](https://learn.freecodecamp.org/machine-learning-with-python/tensorflow)
+- [How Neural Networks Work](https://learn.freecodecamp.org/machine-learning-with-python/how-neural-networks-work)
+  <br />
+  <br />
+  **Projects**: Rock Paper Scissors, Cat and Dog Image Classifier, Book Recommendation Engine using KNN, Linear Regression Health Costs Calculator, Neural Network SMS Text Classifier
+  
+#### Full Stack Development and Python Certifications
 
-Once you have earned all 6 of these certifications, you'll be able to claim your freeCodeCamp.org Full Stack Development Certification. This final distinction signifies that you’ve completed around 1,800 hours of coding with a wide range of web development tools.
+Once you have earned 6 of these Front End Web Development skills and Back End API design certifications, you'll be able to claim your freeCodeCamp.org Full Stack Development Certification along with Additional Certifications in Python, in the emerging field of Data Science and Machine Learning. This final distinction signifies that you’ve completed around 3,000 hours of coursework.
```

Comment:
```suggestion
Once you have earned the Responsive Web Design, Algorithms and Data Structures, Front End Libraries, Data Visualization, APIs and Microservices, and Legacy Information Security and Quality Assurance certifications, you'll be able to claim your freeCodeCamp.org Full Stack Development Certification. This distinction signifies that you've completed around 1,800 hours of coding with a wide range of web development tools.
```

I think we should specifically mention the requirements for the Full Stack certification, and remove the mention of the python certifications.